static/customview.js

```diff
@@ -110,6 +121,9 @@ function updateMCHCombinedBoxPlot(mmu_gid, hsa_gid) {
 function updateGeneElements() {
     var geneSelected = $('#geneName option:selected').val();
     if (geneSelected != 'Select..') {
+        if (typeof(Storage) !== 'undefined') {
+            localStorage.setItem(species, $('#geneName option:selected').text());
```

Comment:
It would be much easier if orthologs always have the same names (excluding lower-/uppercase differences) - is this the case?